designsafe/static/scripts/data-depot/components/data-depot-toolbar/data-depot-toolbar.component.html

```diff
@@ -16,7 +16,7 @@
                 <input class="form-control" style="width:240px;" placeholder="Find in {{ $ctrl.placeholder() }}" ng-model="$ctrl.search.queryString">
                 <button class="btn btn-default" style="clear: right;"> <i class="fa fa-search"></i> </button>
             </form>
-            <div class="btn-group">
+            <div class="btn-group" style="margin-left:0px">
```

Comment:
Can you move this to a CSS class?
We're trying to move away from inline styles.

Reply:
Done.